amdgpu-monitor

```diff
@@ -112,7 +112,7 @@ def main():
         print("Invalid value for sleep specified.  Must be an integer great than zero")
         sys.exit(-1)
 
-    # 
+    # You don't want to call sys.exit(0) for below two lines of code?
     if args.loop == True and args.gui == True: print("--loop and --gui are mutually exclusive")
```

Comment:
Since --loop and --gui are mutually exclusive, do you want to call exit(0) if users set both parameters as True?

Reply:
I thought about it, but decided just to assume the gui option since it is a super set of the 2.  Maybe I should modify the warning?